src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -121,12 +123,19 @@ public function enable_widget_copy_paste( $result, $server, $request ) {
 		$route = $request->get_route();
 		
 		// Check if this matches our target endpoint.
-		if ( ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/encode$#', $route, $matches ) ) {
+		if (
+			! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/encode$#', $route, $matches_encode )
+			&& ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/render$#', $route, $matches_render )
+		) {
 			return $result;
 		}
 	
 		// Get the widget type ID from the route.
-		$widget_type_id = $matches[1];
+		$widget_type_id = $matches_encode[1];
+
+		if ( $matches_render ) {
+			$widget_type_id = $matches_render[1];
+		}
```

Comment:
Along with the above change, you can revert this back to:

```suggestion
		$widget_type_id = $matches[1];
```